### DISCUSSION TITLE
search engine discoverability

### DISCUSSION THREAD

**Discussion by kaciccone** [knowledgefutures/pubpub#2273] [Support]
On August 8 created a new community for "The Long View Project" with the URL https://ncsu-longview.pubpub.org/. We've noticed that it's still not discoverable in a Google search, even though it's been several weeks. Can you help us verify ownership of the site, so it will get indexed? I was trying to follow the steps here to fix the problem: https://support.google.com/webmasters/answer/7474347?hl=en

**Comment by gabestein** [comment]
Hi Karen - I went ahead and submitted it for indexing. It's telling me to check back in a few days, so I'll let you know if anything comes up. My guess is that it hasn't yet been indexed because, although the community may be a few weeks old, the content was only made public on September 23rd. As the document you linked to mentioned, it can take a week for sites to be crawled.
